src/main/java/com/checkout/common/PaymentSourceType.java

```diff
@@ -39,6 +39,23 @@
     @SerializedName("id")
     SEPA,
     @SerializedName("sofort")
-    SOFORT;
-
+    SOFORT,
+    @SerializedName("qpay")
+    QPAY,
+    @SerializedName("alipay")
+    ALIPAY,
+    @SerializedName("paypal")
+    PAYPAL,
+    @SerializedName("multibanco")
+    MULTIBANCO,
+    @SerializedName("eps")
+    EPS,
+    @SerializedName("poli")
+    POLI,
+    @SerializedName("P24")
```

Comment:
Serialized name is `p24`, not `P24`. Please review the list and make it 100% equal to the .NET version.